pom.xml

```diff
@@ -223,5 +223,4 @@
       </plugin>
     </plugins>
   </reporting>
-
```

Comment:
nit: do we need this?

Reply:
No, that was a left-over from another change that was not needed. Removed.